src/models/request-presentation.ts

```diff
@@ -60,7 +58,7 @@ export class RequestPresentation {
           message: request.message,
           createdAt: request.createdAt.getTime(),
           updatedAt: request.updatedAt.getTime(),
-          scheduledAt: awsCronParser.next(cron, new Date()),
+          scheduledAt: new Date(Date.now() + this.schedulerIntervalMS),
```

Comment:
is this really right?  Isn't the schedulerInterval just how often it checks, not how often anchors will actually happen?  I feel like we might just want to remove `scheduledAt` entirely as there's now no way to predict exactly when an anchor will happen now that we're scheduling it based on usage rather than a fixed interval.

@oed

Reply:
(to be clear, we should file a ticket to remove `scheduledAt`, not do it as part of this PR)

Reply:
I concur.